<launch>
  <arg name="gui" default="false" />

  <node pkg="capra_filters" name="manager" type="capra_filters_manager.py" output="screen">
    <param name="nodelet_manager_name" value="nodelet_manager" />
  </node>

  <node pkg="nodelet" name="nodelet_manager" type="nodelet" args="manager" output="screen" />

  <group if="$(arg gui)">
    <node pkg="rqt_gui" name="capra_filters_rqt" type="rqt_gui" />
  </group>
</launch>
